src/recipe/recipe.controller.ts

```diff
@@ -20,8 +20,10 @@ import {
   UpdateRecipeRequest,
 } from './dto';
 import { JwtAuthGuard } from 'src/auth/guards/jwt-auth.guard';
+import { ApiTags } from '@nestjs/swagger';
 
 @Controller('recipe')
+@ApiTags('recipe')
```

Comment:
- route should be in plural form - `recipes`
- API tag should be e.g. `Recipes`

Reply:
I think you can leave it as `Auth` 👍🏻 These tags are cosmetics really - make it so that it looks good in the swagger docs page. Just thought that if it starts with a capital letter it will look better :)